src/Compiler/TypedTree/TypedTree.fs

```diff
@@ -5292,16 +5292,16 @@ type CcuData =
 
     override x.ToString() = sprintf "CcuData(%A)" x.FileName
 
-type CcuTypeForwarderTree<'TKey, 'TValue> =
+type CcuTypeForwarderTree =
     {
-        Value : 'TValue option
-        Children : ImmutableDictionary<'TKey, CcuTypeForwarderTree<'TKey, 'TValue>>
+        Value : Lazy<EntityRef> option
```

Comment:
As an aside - I need to think about whether `Lazy<EntityRef>` is right here now.  `EntityRef` is a delayed-resolution data structure resolved by mutating fixup in any case.  The `Lazy` thus seems superfluous (and `Lazy` are not free).  I'll take a look at whether it can be removed.